docs/rule/require-splattributes.md

```diff
@@ -0,0 +1,62 @@
+# require-splattributes
+
+Splattributes (`...attributes`) make it possible to use attributes on component
+invocations (e.g. `<SomeComponent class="blue">`). Forgetting to add
+`...attributes` however makes it impossible to apply attributes like `class` to
+a component.
+
+This rule warns about templates that don't have `...attributes` in them.
+
+Please note that this rule is only useful for Glimmer components or tagless
+(`tagName: ''`) classic components, because regular classic components have
+this functionality built into the root element, which is not part of their
+templates.
+
+This rule also should not be used foo route/controller templates, because those
+don't support `...attributes`. Instead of unconditionally enabling this rule in
+your `.template-lintrc.js` file, you might want to consider using
+[`overrides`](../overrides.md) to only enable it for component templates.
```

Comment:
Maybe make another section for "recommended configuration" and include an example of how to apply it (e.g. the syntax folks should use in `.template-lintrc.js`)?

Reply:
sorry for the long delay here. I've added an "Example configuration" section.